src/twisted/mail/imap4.py

```diff
@@ -5196,6 +5196,13 @@ def getOtherNamespaces(self):
         return None
 
 
+    def getUserNamespaces(self):
+        """
+        @see: L{INamespacePresenter.getUserNamespaces}
+        """
+        return None
```

Comment:
I'm not sure.  Maybe this interface should have been implemented to return an empty tuple instead of None, but I didn't want to change the semantics of the function, as defined by the interface.

Reply:
True. Thanks. This should have the optional type for sure. Sorry for the noise.